templates/evaluations/evaluation_config_form.html

```diff
@@ -2,14 +2,32 @@
 {% load form_tags i18n %}
 
 {% block form %}
-  <div>
+  <div x-data="{  runGeneration: {{ form.run_generation.value|yesno:'true,false' }}  }">
     {{ form.non_field_errors }}
 
     {% render_form_fields form "name" %}
     {% render_form_fields form "dataset" %}
     {% render_form_fields form "evaluators" %}
 
     <div class="fieldset w-full">
+      <label class="label font-bold" for="">
+        <div>Generation Configuration</div>
+      </label>
+
+      <div id="id_evaluators">
+        <div>
+          <label for="id_run_generation">
+            <input class="checkbox " type="checkbox" name="run_generation" id="id_run_generation"
+                   {% if form.run_generation.value %}checked="checked"{% endif %}
+                   x-model="runGeneration">
+            Run generation step before evaluation
+          </label>
+        {{ form.run_generation.errors }}
+        </div>
+      </div>
+    </div>
```

Comment:
You should be able to do this without with the Django form renderer:

```
{% render_field form.name xmodel="runGeneration" %}
```

Alternatively in the python form you can set attributes on the widget e.g. https://github.com/dimagi/open-chat-studio/blob/33f64319ceb4759d63146f490c34276dfbfd01d1/apps/channels/forms.py#L281-L283

(confusingly, `control_attrs` is for the fieldset: https://github.com/dimagi/open-chat-studio/blob/1bd42f8d617627ddcaf00a614eddde065a7d8253/templates/web/form/text.html#L1)